Modules/System/Cryptography Management/src/DESCryptoServiceProviderImpl.Codeunit.al

```diff
@@ -40,30 +40,30 @@ codeunit 1382 "DESCryptoServiceProvider Impl."
     end;
 
     [NonDebuggable]
-    procedure EncryptStream(Password: Text; InputInstream: InStream; VAR OutputOutstream: Outstream)
+    procedure EncryptStream(Password: Text; Salt: Text; InputInstream: InStream; VAR OutputOutstream: Outstream)
```

Comment:
As this is an internal implementation codeunit, you are free to change function signatures. Just ensure, that the behavior of the function doesn't change, when called by the public facade codeunits.

Reply:
It does, they are both text parameters and when you call the new function with the "Salt" paramater you could basically fill both parameters with the same text for the function to continue working the way it did.